src/binary/ibc-relayer/setup-prometheus.ts

```diff
@@ -0,0 +1,60 @@
+import http from 'http';
+
+import client from 'prom-client';
+
+import { Logger } from '../create-logger';
+
+let initialized = false;
+
+function getMetrics() {
+  return {
+    pollCounter: new client.Counter({
+      name: 'poll_counter',
+      help: 'Poll counter',
+    }),
+  };
+}
+
+export type Metrics = {
+  pollCounter: client.Counter<string>;
+} | null;
+export function setupPrometheus({
+  enabled,
+  port,
+  logger,
+}: {
+  enabled: boolean;
+  port: number;
+  logger: Logger;
```

Comment:
Logger is not yet used. Do you plan to use it in this PR or a direct follow up?
If not, can we remove it until there is a clear use? We can always add it back then

Reply:
It is used here: https://github.com/confio/ts-relayer/pull/173/files#diff-95ad111838746b4a0030c8e37ff66c5de9df11b1a2eab5aa3f59895c9f70016cR57

or do you think it's better to do console log? I would rather keep the logger to make sure the endpoint is mentioned in the logs.